app/views/state_file/questions/md_permanently_disabled/edit.html.erb

```diff
@@ -37,7 +35,7 @@
         <div class="white-group">
           <div class="question-with-follow-up">
             <div class="question-with-follow-up__question">
-                <% show_disability_warning =  !current_intake.filing_status_mfj? ? "#disability-warning" : nil %>
+                <% show_disability_warning =  !current_intake.filing_status_mfj? && could_qualify_for_pension_exclusion ? "#disability-warning" : nil %>
```

Comment:
this followup will never be shown unless the `intake.could_qualify_for_pension_exclusion?`, right? seems redundant but also fine as-is

Reply:
+1 then you can also use the same variable from above.